content/applications/finance/payment_providers.rst

```diff
@@ -38,15 +38,15 @@ subscriptions with recurring payments using their favorite payment methods, such
    card numbers) is stored on Odoo servers or Odoo databases hosted elsewhere. Instead, Odoo apps
    use a unique reference number for the data stored safely in the payment providers' systems.
 
-.. _payment_providers/supported_providers:
+.. _payment-providers-supported-providers:
```

Comment:
Why? The `/` was there to scope the ref, but without it it doesn't make much sense to have a ref named `something-supported-something`.

Reply:
It seems we can't use `/` in doc links in Odoo if we want to point to a custom anchor. Technically, it would be `AppName-FileName-something`, but I guess payment providers are a bit of a special case.

Reply:
Ok, I see. The recommendation must be because HTML anchors are computed based on the heading's title whereas RST anchors are identical to whatever you typed in the sources.

I disagree though, because:
- Most people will rely on HTML anchors when creating documentation links in Odoo with the `documentation=` attribute, thus completely ignoring the carefully crafted RST anchor.
- RST anchors are very often updated when the corresponding heading titles are updated, thus canceling the only pro argument for RST anchors.
- Clearing scoping RST anchors is important because they are sometimes referenced without context from other documentation files and because they might be conflicting anchor names across the documentation. I think the usage of `/` in RST anchors largely outweighs the benefits (if any) of dashed RST anchors.

Reply:
I'm not 100% sure anymore. I don't think I had a problem with the `/` but rather with letters written in capital letters in the code.
@AntoineVDV @xpl-odoo @auva-odoo